proofs/tree.go

```diff
@@ -501,6 +722,30 @@ func getIndexOfString(slice []string, match string) (index int, err error) {
 	return index, fmt.Errorf("getIndexOfString: No match found")
 }
 
+func GetFieldOfStruct(obj interface{}, name string) (interface{}, error) {
```

Comment:
Should this be public?

Reply:
I do not think it should, I will move it private